oxenss/storage/database.hpp

```diff
@@ -32,6 +33,13 @@ enum class StoreResult {
     Full,      // Can't insert right now because the database is full.
 };
 
+inline std::atomic<int> tmp_init_db_version = 0;
```

Comment:
Mutable global variables like this are, generally, not acceptable.  As a general rule, there might be a justification for doing so, but any such justification would need to be compelling and well commented.  This particular instance is unjustifiable: this is simply tracking a database property that was assigned the last time a Database was initialized; that should be in the database object, not a global variable.